core/domain/exp_services.py

```diff
@@ -1070,8 +1362,14 @@ def delete_demo(exploration_id):
 def load_demo(exploration_id):
     """Loads a demo exploration.
 
-    The resulting exploration will have two commits in its history (one for its
-    initial creation and one for its subsequent modification.)
+    The resulting exploration will have two commits in its history (one for
+        its initial creation and one for its subsequent modification.)
```

Comment:
Since this is part of the top-level docstring, deindent this line by 4. You only need the indentation when it needs to be separated from the next item in a list (e.g. a list of args).